openshift/template.yaml

```diff
@@ -130,9 +130,9 @@ parameters:
 - name: JOBS_AUTO_REPORT_LOG_LEVEL
   value: "INFO"
 - name: JOBS_AUTO_REPORT_SUSPEND
-  value: "true"
+  value: "false"
```

Comment:
For the future, please parametrize these values, so they can be changed by app-interface. This will give 2 advantages:
- we can apply the new parameter without the need of upgrading to the latest code version
- we can apply the new change just once, instead of one in the code and one for upgrade in the app-interface manifest